mobile-app/src/screens/ArtworkScreen.tsx

```diff
@@ -39,6 +39,7 @@ const mapState = (state: RootState) => ({
   artworkList: state.artwork.list,
   isLoadingArtwork: state.artwork.isLoading,
   didErrorOccurLoadingArtwork: state.artwork.isError,
+  search: state.artwork.searchQuery,
```

Comment:
If we change `search` to `searchQuery` in the other file, it'd make sense to change it in this one, as well